statistics.md

```diff
@@ -107,6 +109,10 @@ You can perform full collection using the following syntax.
     ANALYZE TABLE TableName PARTITION PartitionNameList INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH|SAMPLES];
     ```
 
+> **Note:**
+>
+> To ensure the consistency of the statistical information both before and after, when you set `tidb_analyze_version=2`, `ANALYZE TABLE TableName INDEX` will also collect statistics of the whole table instead of the given index.
```

Comment:
@winoros 请帮忙看看是说`为了保证收集前后统计信息的一致性`吗？谢谢~
```suggestion
> To ensure that the statistical information before and after the collection is consistent, when you set `tidb_analyze_version=2`, `ANALYZE TABLE TableName INDEX` will also collect statistics of the whole table instead of the given index.
```